platform/integrations/rancher/rancher-integration.mdx

```diff
@@ -10,54 +10,99 @@ import NavStep from '@site/src/components/NavStep'
 import Button from '@site/src/components/Button'
 import Label from '@site/src/components/Label'
 
-After [configuring the Rancher integration](./configuration), you can now create and delete virtual clusters from the Rancher UI. Additionally, you can also access the vCluster Platform directly from the Rancher UI.
+import PartialRancher from "@site/platform/api/_partials/resources/projects/spec/rancher.mdx";
+
+Once Rancher is configured on vCluster Platform, virtual clusters can be imported into Rancher and started using as a cluster in Rancher. In order to enable this integration on each virtual cluster, the vCluster Platform project must be enabled with the Rancher integration first. 
+
+## Enable vCluster Platform project with the Rancher Integration
```

Comment:
Do you think we can add this instead? `Enable Rancher Integration on vCluster Platform Project`

Reply:
yes